Add: new blog post - czi funding announcement

lwasser opened the pull request:
NOTE: This blog post needs to be renamed before merging so we can use the live preview feature setup here. 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205942179869614
  - https://app.asana.com/0/0/1206246014367555

lwasser committed:
Add: new blog post - czi

lwasser committed:
Fix: htmlproofer test

Fix: install gems locally

Fix: install gems locally

Fix: install gems locally

Fix: another try

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+
+### Broad inclusion is critical for both open source and open science
+
+Broad inclusion of individuals in the open source ecosystem which has been found to be [even less diverse then the broader tech community](https://www.wired.com/2017/06/diversity-open-source-even-worse-tech-overall/) is also critical to pyOpenSci's mission. pyOpenSci wants to empower everyone to participate in both the development and use of open source software. This
```
```suggestion
Broad inclusion of underrepresented and historically excluded individuals is critical to pyOpenSci's mission. The open source ecosystem has been found to be [even less diverse then the broader tech community], (https://www.wired.com/2017/06/diversity-open-source-even-worse-tech-overall/) and to help remedy this, pyOpenSci empowers everyone to participate in both the development and use of open source software. This
```

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+empowerment enables open science to reach its full potential.
+
+### Open source maintainers need more support
+Despite the importance of open source software and inclusion to fundamental
```
I'm not entirely clear what we're trying to say here. What's throwing me is the phrase "and inclusion"

lwasser replied:
i hear you. I think the rewrite goal was to talk about 
1. how open source is important to open science
2. inclusion is important to both 

maybe there is a way to make that more clear? there are papers about the importance of dei in science and team work in general. but thing link of ideas is a bit tricky

lwasser replied:
ok i implemented a minor fix and just dropped the inclusion as it really didn't make sense there!

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+### Open source maintainers need more support
+Despite the importance of open source software and inclusion to fundamental
+open science principles, open source maintainers do not get the support they
+need. Maintainers need both institutional and community support in learning
```
```suggestion
Maintainers need both institutional and community support in learning
and engaging in open source software development, as well as in maintaining tools and interacting
with the broad user base that uses these tools in support of open science.
```

lwasser replied:
can you help me understand the intended change here @kierisi so i can apply it?

kierisi replied:
oh! yes, for some reason it wouldn't let me select more than a single line for review. so what's showing up in green are my suggested changes, but it's not showing up as a diff

lwasser replied:
oh interesting! maybe we can look at that together to see why :) sometimes the github gui can be a bit quirky!

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+
+## pyOpenSci is pushing for change
+
+pyOpenSci is pushing for change. We envision a world where:
```
```suggestion
We envision a world where:
```

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+Broad inclusion of individuals in the open source ecosystem which has been found to be [even less diverse then the broader tech community](https://www.wired.com/2017/06/diversity-open-source-even-worse-tech-overall/) is also critical to pyOpenSci's mission. pyOpenSci wants to empower everyone to participate in both the development and use of open source software. This
+empowerment enables open science to reach its full potential.
+
+### Open source maintainers need more support
```
```suggestion
### Open source maintainers need support
```

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+
+pyOpenSci is pushing for change. We envision a world where:
+
+* the scientific Python open source ecosystem is diverse and inclusive,
```
```suggestion
* The scientific Python open source ecosystem is diverse and inclusive.
```

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+pyOpenSci is pushing for change. We envision a world where:
+
+* the scientific Python open source ecosystem is diverse and inclusive,
+* individuals developing open software are supported and
```
```suggestion
* Individuals developing open software are supported.
```

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+
+* the scientific Python open source ecosystem is diverse and inclusive,
+* individuals developing open software are supported and
+* scientists have the skills needed to both contribute to open source and also develop collaborative open science workflows; these skills have significant overlap.
```
```suggestion
* Scientists have the skills needed to both contribute to open source and also develop collaborative open science workflows; these skills have significant overlap.
```

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+* individuals developing open software are supported and
+* scientists have the skills needed to both contribute to open source and also develop collaborative open science workflows; these skills have significant overlap.
+
+## pyOpenSci is building a supportive and inclusive world through three programs
```
```suggestion
## pyOpenSci builds a supportive and inclusive world through three programs
```

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+    <a href="/images/pyopensci-pillars-flower.png">
+    <img src="/images/pyopensci-pillars-flower.png" style="max-width:100%" alt="A flower petal image with 3 flower petals and a flower center. In each petal there is text. The first says software peer review, the second community partnerships and the third packaging resources and standards. The center circle of the flower says diverse inclusive community.">
+    </a>
+    <figcaption> pyOpenSci is supports scientists developing open source software through three programs: 1) peer review of scientific software, 2) Community partnerships and 3) packaging resources and standards. These three programs are supported by a diverse and inclusive community that cares about the open source software that is needed for open science.
```
```suggestion
    <figcaption> pyOpenSci supports scientists developing open source software through three programs: 1) peer review of scientific software, 2) community partnerships and 3) packaging resources and standards. These three programs are supported by a diverse and inclusive community that cares about the open source software that is needed for open science.
```

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+
+The three programs pyOpenSci runs are:
+
+* [Peer review of scientific Python software](https://www.pyopensci.org/about-peer-review/index.html) - pyOpenSci software peer review process helps scientists find vetted and trusted tools that they need to build reproducible open science workflows. We also are empowering our community with critical open science skills that support contributing to open source software.
```
```suggestion
* [Peer review of scientific Python software](https://www.pyopensci.org/about-peer-review/index.html) - The pyOpenSci software peer review process helps scientists find the vetted and trusted tools they need to build reproducible open science workflows. We also empower our community with critical open science skills that support contributing to open source software.
```

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+
+* [Peer review of scientific Python software](https://www.pyopensci.org/about-peer-review/index.html) - pyOpenSci software peer review process helps scientists find vetted and trusted tools that they need to build reproducible open science workflows. We also are empowering our community with critical open science skills that support contributing to open source software.
+
+* [Community partnerships with domain-specific scientific Python communities](https://www.pyopensci.org/partners.html). Domain-specific communities  partner with PyOpenSci to leverage our peer review process as a way to track vetted high quality tools. They also support development of Python packaging packaging guidelines in an effort to streamline packaging standard development across the scient ...
```
```suggestion
* [Community partnerships with domain-specific scientific Python communities](https://www.pyopensci.org/partners.html). Domain-specific communities partner with PyOpenSci to leverage pyOpenSci's peer review process as a way to track vetted, high quality tools. Communities also support the development of Python packaging packaging guidelines in an effort to streamline the development of packaging standards across the scientific Python ecosystem.
```

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+
+* [Community partnerships with domain-specific scientific Python communities](https://www.pyopensci.org/partners.html). Domain-specific communities  partner with PyOpenSci to leverage our peer review process as a way to track vetted high quality tools. They also support development of Python packaging packaging guidelines in an effort to streamline packaging standard development across the scient ...
+
+* Training and resources to help scientists develop and maintain high-quality, accessible, open source software. Our community developed [Python packaging guide](https://www.pyopensci.org/python-package-guide/) provides resources and tutorials that help scientists both navigate a complex Python packaging ecosystem while also making recommendations for community accepted best practices.
```
```suggestion
* Training and resources to help scientists develop and maintain high-quality, accessible, open source software. Our community developed the [Python packaging guide](https://www.pyopensci.org/python-package-guide/), which provides resources and tutorials that help scientists navigate a complex Python packaging ecosystem. The Python packaging guide also makes recommendations for community accepted best practices.
```

kierisi commented on _posts/2023-12-29-czi-funds-pyopensci-2024.md:
```diff
@@ -0,0 +1,76 @@
+pyOpenSci is excited to grow a more inclusive and supportive scientific Python community in 2024!
+
+We are always looking for volunteers to support our community programs.
+If you are interested in getting involved with pyOpenSci, you can [learn more here](/get-involved-contact.html).
```
```suggestion
If you are interested in getting involved with pyOpenSci, [learn more here](/get-involved-contact.html).
```

kierisi reviewed (commented):
added my thoughts - mostly grammatical and flow suggestions!

lwasser committed:
Fix: edits from Jesse

Co-authored-by: Jesse Mostipak <jesse.maegan@gmail.com>

Update _posts/2023-12-29-czi-funds-pyopensci-2024.md

Co-authored-by: Jesse Mostipak <jesse.maegan@gmail.com>

lwasser committed:
Fix: edits from @kierisi

Co-authored-by: Jesse Mostipak <jesse.maegan@gmail.com>

Apply suggestions from code review

Co-authored-by: Jesse Mostipak <jesse.maegan@gmail.com>

Fix: minor edits after review

Fix: Rename for jan 3 publication

Fix: Rename for jan 3 publication

lwasser commented:
ok!! i think this should be ready to go :) i'll rename locally and we can still accept minor edits with an intended publish date of  next week when we can back from the holiday!

lwasser committed:
Final cleanup

lwasser committed:
Remove the old blue bird

lwasser committed:
Add: note about building with the future flag
